task/06-conditions-n-loops-tasks.js

```diff
@@ -209,7 +245,12 @@ function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
  * 'noon' => 'noon'
  */
 function reverseString(str) {
-    throw new Error('Not implemented');
+    let buffer= '';
+    for( let i = str.length -1; i>=0; i--)
+    {
+    buffer +=str[i];
+    }
+    return buffer;
```

Comment:
Please, try to solve this task using Array method 'reverse'
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse
We can transform string into array  at first.